src/Atomics/kineticionizationdata.manual.cpp

```diff
@@ -0,0 +1,63 @@
+/** 
+ * Template for how a future automatically generated 
```

Comment:
This file is actually the old, manually constructed data file. I decided to keep it in case weird things start to happen along the line so that debugging could be easier.